e2e/playwright-visual.config.js

```diff
@@ -2,30 +2,49 @@
 // playwright.config.js
 // @ts-check
 
-/** @type {import('@playwright/test').PlaywrightTestConfig} */
+/** @type {import('@playwright/test').PlaywrightTestConfig<{ theme: string }>} */
 const config = {
     retries: 0, // visual tests should never retry due to snapshot comparison errors
     testDir: 'tests/visual',
-    timeout: 90 * 1000,
-    workers: 1, // visual tests should never run in parallel due to test pollution
+    testMatch: '**/*.visual.spec.js', // only run visual tests
+    timeout: 60 * 1000,
+    workers: 2, // TODO See if this works
     webServer: {
         command: 'cross-env NODE_ENV=test npm run start',
         url: 'http://localhost:8080/#',
         timeout: 200 * 1000,
         reuseExistingServer: !process.env.CI
     },
     use: {
-        browserName: "chromium",
         baseURL: 'http://localhost:8080/',
-        headless: true, // this needs to remain headless to avoid visual changes due to GPU
+        headless: true, // this needs to remain headless to avoid visual changes due to GPU rendering in headed browsers
         ignoreHTTPSErrors: true,
         screenshot: 'on',
-        trace: 'off',
+        trace: 'on',
         video: 'off'
     },
+    projects: [
+        {
+            name: 'chrome',
+            use: {
+                browserName: 'chromium'
+            }
+        },
+        {
+            name: 'snow-theme-chrome',
```

Comment:
These project names can be human readable (including spaces etc).

Maybe we can update with something like

```
{
    name: 'Chrome (Default Theme)',
    use: {
        browserName: 'chromium'
    }
},
{
    name: 'Chrome (Snow Theme)',
    use: {
        browserName: 'chromium',
        theme: 'snow'
    }
}
```

Reply:
that would be awesome but then our users would have to pass that on the command line :(